### ISSUE TITLE
error in budgetedsvm

### ISSUE THREAD

**Issue by manvirvirk** [jclevesque/bagged-budget-svms#2]
i m getting following error
(esvm) D:\bagged-budget-svms-master>python setup.py install
running install
running build
running build_py
Compiling budgetedsvm toolbox.
********************************************************************************
error: [WinError 2] The system cannot find the file specified
 how to install budgetedsvm? THANKS

**Comment by jclevesque** [comment]
It seems you are using windows. I rely on some Unix tools such as make. You'd probably be better off using a vanilla machine learning model in scikit-learn.

**Comment by manvirvirk** [comment]
yes i m using windows, so is there any option as i don't have linux VM?

**Comment by jclevesque** [comment]
You can get the budgeted svm toolbox embedded in here to compile and then fix the bindings to call it in Windows. But it also depends on why you need this. If you want to compare ensembles of budgeted SVMs it might make sense to try to fix this, but if you just want any ensemble classifier there are many options out there.

**Comment by manvirvirk** [comment]
I want ensemble of SVM classifiers to classify features.

**Comment by jclevesque** [comment]
If your problem is not large scale you can do that with scikit learn, build
a wrapper around the SVM classifier, train multiple SVMs on bagged
subsamples (or subspaces, feature-wise) of the training set. I'd look into
Gradient boosted trees, random forests and XGboost, those are much more
mature now and can work for most use cases.
